### ISSUE TITLE
Links to images broken

### ISSUE THREAD

**Issue by mat813** [AVATEAM-IT-SYSTEMHAUS/mkdocs-kroki-plugin#29]
So, as described in #25, I have a file `backbone/schema.md` that has a kroki graph. the path to the generated image is now correct, but the generated html is not:

```html
<p><img alt="Kroki" src="{'../../images/kroki_generated/schemas-1cf3fe41743e0a1c1567d1e95f6f05e4.svg'}" /></p>
```

Also, mkdocs generates a warning:

```
mkdocs-infra-1  | WARNING  -  Documentation file 'backbone/schemas.md' contains a link to 'backbone/images/kroki_generated/schemas-1cf3fe41743e0a1c1567d1e95f6f05e4.svg'}' which is not found in the documentation files.
```

**Comment by oniboni** [comment]
Hi, I could reproduce the issue in my test environments and wrote a fix.

Can you confirm, that it solves your issues as well?

**Comment by mat813** [comment]
It does not, see https://github.com/AVATEAM-IT-SYSTEMHAUS/mkdocs-kroki-plugin/pull/30#issuecomment-1420910883

**Comment by juanitocalero** [comment]
I'm having the same issue. Looks like there is a problem with the last release.  I can confirm that it works with version 0.5.2

**Comment by oniboni** [comment]
It should now work as expected. #30

**Comment by mat813** [comment]
@oniboni yes, now it works :smiley:

**Comment by b-bittner** [comment]
Thanks for all your support!
And sorry for the delay..

**Comment by b-bittner** [comment]
I've merged #27 into main, but not yet as the newest release.
Maybe someone can confirm ist fixing the issue and and not creating new ones.
